db/schema.rb

```diff
@@ -1297,12 +1303,6 @@
     t.datetime "updated_at",             null: false
   end
 
-  create_table "web_sections", force: :cascade do |t|
```

Comment:
Why did you "rollback" the database? :thinking:

Reply:
it occured by mistake